js/app.js

```diff
@@ -0,0 +1,483 @@
+	// it was originally defined as. The prototype thing didn't, but the
+	// way that I wrote it did.
+
+console.log(myArray);
+
+// F. Biggie Smalls
+
+const specificNumber = 7
+
+// 1.
+if(specificNumber < 5) {
+	console.log("Little number...");
+// 2.
+} else if (specificNumber > 10) {
+	console.log("Big number!")
+}
+
+// G. Monkey in the Middle
+
+if(specificNumber < 5) {
+	console.log("Little number...")
+} else if(specificNumber > 10) {
+	console.log("Big number!");
+} else {
+	console.log("Monkey...")
+}
+
+// What is in Your Closet?
+
+const kristynsCloset = [
+  "left shoe",
+  "cowboy boots",
+  "right sock",
+  "GA hoodie",
+  "green pants",
+  "yellow knit hat",
+  "marshmallow peeps"
+];
+
+// Thom's closet is more complicated. Check out this nested data structure!!
+const thomsCloset = [
+  [
+    // These are Thom's shirts
+    "grey button-up",
+    "dark grey button-up",
+    "light blue button-up",
+    "blue button-up",
+  ],[
+    // These are Thom's pants
+    "grey jeans",
+    "jeans",
+    "PJs"
+  ],[
+    // Thom's accessories
+    "wool mittens",
+    "wool scarf",
+    "raybans"
+  ]
+];
+
+
+// 1.
+
+console.log("Kristyn is rocking that " + kristynsCloset[2] + " today!");
+
+// 2.
+kristynsCloset.splice(6, 0, "raybans");
+
+// 3.
+kristynsCloset[5] = "stained knit hat"
+
+// 4.
+thomsCloset[0][0];
+
+// 5.
+thomsCloset[1][2];
+
+// 6.
+thomsCloset[2][1]
+
+// 7.
+console.log("Thom is looking fierce in a " + thomsCloset[0][0] + ", " + thomsCloset[1][2] + ", and " + thomsCloset[2][1] + "!")
+
+// 8.
+
+thomsCloset[1][2] = "Footie Pajamas"
+
+// printGreeting
+
+// Again, well played!
+
+// printCool
+
+const printCool = (name) => {
+	console.log(name + " is cool");
+}
+printCool('Chris')
+
+// calculateCube
+
+const calcCube = (num) => {
+	return num ** 3;
+}
+console.log(calcCube(5))
+
+//isVowel
+
+const isAVowel = (letter) => {
+	lowercase = letter.toLowerCase()
+	if(lowercase === 'a' || lowercase === 'e' || lowercase === 'i' || lowercase === 'o' || lowercase === 'u') {
+		return true;
+	} else if (lowercase === 'y') {
+		return 'I mean kinda...'
+	} else {
+		return false;
+	}
+}
+
+console.log(isAVowel('Y'))
+
+// getTwoLengths
+
+const getTwoLengths = (arr1, arr2) => {
+	const arraysLength = []
+	arraysLength.push(arr1.length, arr2.length)
+	return arraysLength;
+}
+
+console.log(getTwoLengths("What", "Yea"));
+
+// getMultipleLengths
+
+const getMultLengths = (str) => {
+	let lengthStorage = [];
+	for(let i = 0; i < str.length; i++) {
+		let breakDown = str[i];
+		lengthStorage.push(breakDown.length);
+	}
+	return lengthStorage;
+}
+
+console.log(getMultLengths(["Hey", "Yeah", "Super", "Totally"]));
+
+// printLongestWord
+
+const findLongestWord = (arrayOne) => {
+	let longestWord = '';
+	arrayOne.forEach(function(words) {
+		if(words.length > longestWord.length) {
+			longestWord = words
+		}
+	});
```

Comment:
as we discussed after you submitted this, use singular noun for parameter of the callback.   (semantics are important)